pkgs/tools/games/opentracker/default.nix

```diff
@@ -1,38 +1,43 @@
-{ lib
-, stdenv
-, buildDotnetModule
-, fetchFromGitHub
-, autoPatchelfHook
-, wrapGAppsHook
-, dotnetCorePackages
-, fontconfig
-, gtk3
-, openssl
-, libX11
-, libXi
-, xinput
+{
+  lib,
+  stdenv,
+  buildDotnetModule,
+  fetchFromGitHub,
+  autoPatchelfHook,
+  wrapGAppsHook,
+  dotnetCorePackages,
+  fontconfig,
+  gtk3,
+  libunwind,
+  openssl,
+  xinput,
+  xorg,
 }:
-
 buildDotnetModule rec {
   pname = "opentracker";
-  version = "1.8.2";
+  version = "1.8.5";
 
   src = fetchFromGitHub {
     owner = "trippsc2";
     repo = pname;
-    rev = version;
-    sha256 = "0nsmyb1wd86465iri9jxl3jp74gxkscvnmr3687ddbia3dv4fz0z";
+    rev = "refs/tags/${version}";
+    hash = "sha512-nWkPgVYdnBJibyJRdLPe3O3RioDPbzumSritRejmr4CeiPb7aUTON7HjivcV/GKor1guEYu+TJ+QxYrqO/eppg==";
   };
 
-  dotnet-runtime = dotnetCorePackages.runtime_3_1;
-  dotnet-sdk = dotnetCorePackages.sdk_3_1;
+  patches = [./remove-project.patch];
+
+  dotnet-runtime = dotnetCorePackages.runtime_6_0;
```

Comment:
You should also set dotnet-sdk field to net6, by default it's net7, and building with a different dotnet version can cause problems

Reply:
The default is currently .NET 6

https://github.com/NixOS/nixpkgs/blob/690eb0d43e51ec138454c2d6b689d7b697a3399f/pkgs/build-support/dotnet/build-dotnet-module/default.nix#L79-L80

It shouldn't matter, unless the project is using reflection, and expecting some specific assembly implementation detail. In which case, they should specify it using `global.json`.